src/main/webapp/js/email-browser.js

```diff
@@ -19,6 +19,74 @@
 var jasig = jasig || {};
 
 (function($, fluid) {
+
+	var ischecked = 0;
```

Comment:
> I did not know how to name simply `the counting of the checkboxesGlobalChecknessState`

A little long, but you pretty much got it. :+1:  :smile: :gb: 
`checkboxCounter` could be shorter alternative.